### ISSUE TITLE
Why does ShareX include Change DNS function?

### ISSUE THREAD

**Issue by gqqnbig** [ShareX/ShareX#3494]
Although DNS Changer is a convenient tool, isn't ShareX positioned as an image capturing tool? Changing DNS has no direct relation to capturing images or uploading images.

Admittedly uploading images or even accessing website requires DNS lookup, changing DNS is a totally separated function. Users can change DNS using Windows itself or a command line.

In my humble opinion, DNS Changer is not part of ShareX. For developers to reduce maintenance cost, for end users to have a smaller program size and lower memory usage, can you remove DNS Changer?

**Comment by gqqnbig** [comment]
@Jaex No, no, I didn't understand the background but now it totally makes sense. I was born in a much more heavily censored country, where the trick of changing DNS doesn't work for most foreign websites, including facebook, twitter, youtube, etc. In the past people could buy VPN or SSH proxies but nowadays using VPN or SSH itself without consent of government is a crime.

I clearly empathize with you guys and support censorship circumvention, and wish you the best.

**Comment by shikamaruzaza** [comment]
When has to my wife come back in to the home with me. In time to night on
day. I has for delete with all user and do not mess with you forever. My
wife she not come to home with me in the 08.00 pm. It let me die. I’m
willing and in the time to die. You will olso be in charge. Because I have
all the information about you. I don’t messig about. I want to my wife come
home in Utt. That is we end it.

**Comment by eth** [comment]
Man I used this DNS changer tool all the time...
Thanks for removing a perfectly good tool that was very useful!
